### ISSUE TITLE
File Sync Option Does Not Use "Backup Instance Name" To Separate Sync'd Files From Different Systems

### ISSUE THREAD

**Issue by BrettEBowman** [lavinir/hassio-onedrive-backup#123]
When using the "File Sync" option, in OneDrive, the sync'd files from all systems are getting mixed together (and overwritten where duplicates exist?) under the hassio_onedrive_backup/FileSync/ folder. I would expect that the "Backup Instance Name" parameter should be used as the "top level folder" here so that files from each system are kept separate.

**Comment by lavinir** [comment]
You're right, the instance name only effects the backups (as implied by the name..) 
It makes sense to have that ability for File Syncs as well. I will add this to the backlog

**Comment by BrettEBowman** [comment]
I would like to see this ability added. It is confusing (and problematic when there are duplicate file names) with the files from multiple HA servers getting mashed together.
